phpBB/includes/mcp/mcp_logs.php

```diff
@@ -140,7 +140,7 @@ function main($id, $mode)
 						$conditions['topic_id'] = $topic_id;
 					}
 
-					$phpbb_log->delete('mod', $conditions);
+					//$phpbb_log->delete('mod', $conditions);
```

Comment:
If you comment out code at least add a comment, or just remove the code, that's what version control systems are for after all ;-)